tools/scripts/ansible/hosts

```diff
@@ -2,9 +2,15 @@
 cldr-smoke.unicode.org
 cldr-staging.unicode.org
 
+[staging:vars]
+ddenv=staging
```

Comment:
a comment that "dd" stands for DataDog would be nice; also underscore dd_env would make this consistent with dd_key and dd_site (though "Foolish consistency is the hobgoblin of little minds")